cmd/qat_plugin/README.md

```diff
@@ -52,81 +228,84 @@ The number of devices discovered are:10
+
+```bash
+$ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes/demo
+$ ./build-image.sh crypto-perf
+...
+Successfully tagged crypto-perf:devel
 ```
 
-**Note**: It is also possible to run the QAT device plugin using a non-root user. To do this,
-the nodes' DAC rules must be configured to allow PCI driver unbinding/binding, device plugin
-socket creation and kubelet registration. Furthermore, the deployments `securityContext` must
-be configured with appropriate `runAsUser/runAsGroup`.
+#### Deploy the pod
 
-### Verify QAT device plugin is registered on master:
-```
-$ kubectl describe node <node name> | grep qat.intel.com/generic
- qat.intel.com/generic: 10
- qat.intel.com/generic: 10
-```
+In the pod specification file, add container resource request and limit.
+For example, `qat.intel.com/generic: <number of devices>` for a container requesting QAT devices.
 
-### Consuming QAT device plugin
+For a DPDK-based workload, you may need to add hugepage request and limit.
 
-1. Build the DPDK image:
+```bash
+$ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
+$ kubectl apply -k deployments/qat_dpdk_app/base/
+$ kubectl get pods
+  NAME                     READY     STATUS    RESTARTS   AGE
+  qat-dpdk                 1/1       Running   0          27m
+  intel-qat-plugin-5zgvb   1/1       Running   0          3h
 
-     ```
-     $ cd demo
-     $ ./build-image.sh crypto-perf
-     ```
+```
+
+> **Note**: The deployment example above uses kustomize that is available in
+> kubectl since Kubernetes v1.14 release.
 
-     This command produces a Docker image named `crypto-perf`.
+#### Run the test
 
-2. Deploy a pod to run an example DPDK application named
-`dpdk-test-crypto-perf`.
+Manually execute the `dpdk-test-crypto-perf` application to review the logs:
 
-      In the pod specification file, add container resource request and limit.
-      For example, `qat.intel.com/generic: <number of devices>` for a container requesting QAT devices.
+```bash
+$ kubectl exec -it qat-dpdk bash
 
-      For a DPDK-based workload, you may need to add hugepage request and limit.
+$ ./dpdk-test-crypto-perf -l 6-7 -w $QAT1 -- --ptest throughput --\
+devtype crypto_qat --optype cipher-only --cipher-algo aes-cbc --cipher-op \
+encrypt --cipher-key-sz 16 --total-ops 10000000 --burst-sz 32 --buffer-sz 64
 
-     ```
-     $ kubectl apply -k deployments/qat_dpdk_app/base/
-     $ kubectl get pods
-       NAME                     READY     STATUS    RESTARTS   AGE
-       qat-dpdk                 1/1       Running   0          27m
-       intel-qat-plugin-5zgvb   1/1       Running   0          3h
+```
 
-     ```
+### OpenSSL QAT demo
 
-    **Note**: The deployment example above uses kustomize that is available in
-    kubectl since kubernetes v1.14 release.
+> **FIXME** Add details, possibly around how to use the
+> [QAT OpenSSL screencast script](../../demo/screencast-qat.sh) - does it have any requirements?
```

Comment:
the requirements are basically the Kata doc. In order to use OpenSSL QAT, we still need the out-of-tree drivers. With the Kata setup, the host can use the upstream drivers and the QAT device plugin runs OK (this README). The workload needs the out-of-tree driver but that is installed in Kata VM (instructions in the Kata doc).

Reply:
OK, I've just called out to the kata docs.... don't want to duplicate, and the info is over there already etc.